### PR TITLE
Db script

### PR DESCRIPTION
**Pull request by irfamustafa**
This script creates the DB Schema for the project along with all of the table create statements

### CODE REVIEW

**Review by kabollinger03 — APPROVED** [review]

File: DBSchemaTables.sql [hunk 1]
```diff
@@ -0,0 +1,126 @@
+CREATE USER Student_Performance IDENTIFIED BY Student_Performance;
+GRANT CONNECT, RESOURCE, DBA To Student_Performance;
```
Comment (cgreen9008) [inline DBSchemaTables.sql:2]: I like this line right here.

**Review by apennella — APPROVED** [review]

**Review by apennella — APPROVED** [review]

**Review by apennella — APPROVED** [review]
Ya done good kid

**Review by kjones2296 — APPROVED** [review]

**Review by phoongp — APPROVED** [review]

**Comment by kabollinger03** [comment]
Approved by Kate and Kevin

**Comment by kabollinger03** [comment]
Confirmed by Kate and Kevin

**Pull request merged**